.github/workflows/publish.yaml

```diff
@@ -54,11 +54,37 @@ jobs:
       - name: Save package for following jobs
         uses: actions/upload-artifact@v3
         with:
-          name: dist-macos-${{ matrix.python-version }}
+          name: dist-macos-intel-${{ matrix.python-version }}
+          path: dist
+
+  build-macos-arm:
+    runs-on: macos-latest-xlarge
```

Comment:
according to this https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories `latest` points to `macos-12`, but the one using M1 is `macos-14`?

Reply:
I think we can use the large machines, where IIUC correctly large and xlarge are how they are distinguished https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/about-larger-runners#:~:text=The%20following%20machines%20sizes%20are%20available%20for%20macOS%20larger%20runners.